docs/content/v0.20/user-interaction.md

```diff
@@ -0,0 +1,193 @@
+# User Interaction
```

Comment:
This looks very redundant with the material in the examples. Maybe just point to them, maybe with a very brief outline of what the reader should look for there?

Reply:
The main reason I created this file is there is because we are still debating if document by examples or just provide a simple quick start and then document by features. I think we may go with the latter approach in the end, but then we need still to document the user interaction. If this helps, I can point to the example 1 and remove some of the content that overlaps with it  in the user interaction section, as it is likely that we will use example 1 as the "Quick start example"

Reply:
I do not think "document by examples" and "a simple quickstart [presumably plus non-full-example-based documentation]" are the only choices. I think a few examples plus non-full-example-based documentation is also a possibility, and is the path that we are currently on.

Since the material proposed here for this file overlaps so strongly with the existing example 1, I would indeed replace the material here with a pointer to example 1 and, perhaps, a list of points for the reader to look out for in example 1.

Reply:
Sounds good.